faraday.gemspec

```diff
@@ -21,9 +21,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'multipart-post', '>= 1.2', '< 3'
   spec.add_dependency 'ruby2_keywords'
 
-  files = %w[CHANGELOG.md LICENSE.md README.md Rakefile examples lib spec]
-  spec.files = `git ls-files -z #{files.join(' ')}`.split("\0")
-  spec.require_paths = %w[lib spec/external_adapters]
+  spec.files = Dir['CHANGELOG.md', '{examples,lib}/**/*', 'LICENSE.md', 'README.md']
+  spec.require_paths = ['lib']
```

Comment:
careful because the `spec/external_adapters` has been removed and is required for external adapter gems

Reply:
Oh, I tried to check if it was a necessity but couldn't find that, I am sorry. I'll add it right back! 😅